### DISCUSSION TITLE
Music muffled in zoom.

### DISCUSSION THREAD

**Discussion by Supamav** [sircharlo/meeting-media-manager#2142] [General]
We are using the latest MMM, OBS and Zoom versions.

Noticed tonight that the songs in Zoom for the meeting are muffled or somehow muted.  Also music or sound outside of speech in videos is muffled also.  Actual voices are fine.  I'm sick, and unable to be at the meeting this week, but the sound appears to be OK in the Kingdom Hall itself.

Did something possibly change in a recent zoom update?  Can anyone else reproduce and recommend a solution?  I'll try and troubleshoot at the meeting on Sunday to see if something has changed.  Maybe 'Zoom Enhanced Audio' is the culprit?

I'd like to rule out MMM.

Much thanks from Canada,
Jason

**Comment by sircharlo** [comment]
Did you enable original sound in Zoom by curiosity?

**Reply by Supamav** [reply]
I'll have to check that on Sunday.  The latest Zoom update forces the Enhanced Sound option.  That I believe will most likely be the issue, I'll update once I confirm.

Thanks for getting back to me.
